aio/package.json

```
@@ -87,28 +87,28 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "9.0.0",
-    "@angular/cdk": "^9.0.0",
-    "@angular/common": "9.0.0",
-    "@angular/compiler": "9.0.0",
-    "@angular/core": "9.0.0",
-    "@angular/elements": "9.0.0",
-    "@angular/forms": "9.0.0",
-    "@angular/material": "^9.0.0",
-    "@angular/platform-browser": "9.0.0",
-    "@angular/platform-browser-dynamic": "9.0.0",
-    "@angular/router": "9.0.0",
-    "@angular/service-worker": "9.0.0",
+    "@angular/animations": "9.1.0",
+    "@angular/cdk": "^9.1.0",
+    "@angular/common": "9.1.0",
+    "@angular/compiler": "9.1.0",
+    "@angular/core": "9.1.0",
+    "@angular/elements": "9.1.0",
+    "@angular/forms": "9.1.0",
+    "@angular/material": "^9.1.0",
+    "@angular/platform-browser": "9.1.0",
+    "@angular/platform-browser-dynamic": "9.1.0",
+    "@angular/router": "9.1.0",
+    "@angular/service-worker": "9.1.0",
     "@webcomponents/custom-elements": "1.2.1",
     "rxjs": "^6.5.3",
     "tslib": "^1.10.0",
     "zone.js": "~0.10.3"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "0.900.1",
+    "@angular-devkit/build-angular": "0.901.0",
     "@angular/cli": "9.0.1",
```

@atscott whenever you update `build-angular` you should update `@angular/cli` as well because they are released together and interdependent (not a great user experience if you edit these files by hand, that's why `ng update` is recommended way to update Angular projects even for minor versions)

```suggestion
    "@angular/cli": "9.1.0",
```

---

Good catch. Not sure how this one got missed.